maps/NRVHorizon.dmm

```diff
@@ -4723,7 +4685,7 @@
 "bMQ" = (/obj/machinery/door_control{id_tag = "Dorm2"; name = "Dorm 2 Door Bolt Control"; normaldoorcontrol = 1; pixel_x = 0; pixel_y = 24; req_access_txt = "0"; specialfunctions = 4},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layered{dir = 4},/obj/machinery/atmospherics/pipe/simple/supply/hidden/layered{dir = 10},/turf/simulated/floor/wood,/area/crew_quarters/sleep)
 "bMR" = (/obj/structure/table,/obj/machinery/atmospherics/unary/vent_scrubber/on/layered{dir = 8},/obj/machinery/light/small{dir = 1},/turf/simulated/floor/wood,/area/crew_quarters/sleep)
 "bMS" = (/obj/effect/landmark{name = "xeno_spawn"; pixel_x = -1},/turf/simulated/floor/engine,/area/science/hallway)
-"bMT" = (/obj/structure/table/reinforced,/obj/item/weapon/plastique{pixel_x = 3},/obj/item/weapon/plastique,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
```

Comment:
It looks like you just deleted these instead of replacing them with c4, sneaky mapper

Reply:
The map editor to fix this issue is garbo to say the least, but I'll see around to putting it back in...wherever the fuck it was.